UI/cmake/feature-whatsnew.cmake

```diff
@@ -1,9 +1,14 @@
+include_guard(DIRECTORY)
+
 option(ENABLE_WHATSNEW "Enable WhatsNew dialog" ON)
 
 if(ENABLE_WHATSNEW AND TARGET OBS::browser-panels)
   if(OS_MACOS)
     include(cmake/feature-macos-update.cmake)
-  elseif(OS_LINUX)
+  elseif(
+    OS_LINUX
+    OR OS_FREEBSD
+    OR OS_OPENBSD)
```

Comment:
Intentionally expanding based on the answer that BSDs should more or less be on the same level as Linux.

Reply:
Mostly just because this feature isn't super useful outside of official first-party builds, which we currently only provide for Ubuntu and Flatpak. I haven't tested it on other BSDs but I guess it should work?

Reply:
You could argue it doesn't make sense on those platforms because while users might be informed about a new release of OBS Studio, its availability as a system package is dependent on the maintainer of a ports package (if there is any).